light_node/src/file_config.rs

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) 2015 Andrew Gallant
+// Copyright (c) 2019 SimpleStaking and Tezedge Contributors
+// SPDX-License-Identifier: MIT
+
+
+// This module provides routines for reading ripgrep config "rc" files. The
+// primary output of these routines is a sequence of arguments, where each
+// argument corresponds precisely to one shell argument.
+
+use std::error::Error;
```

Comment:
Look at https://github.com/rust-lang-nursery/failure how error handling is done there. We are using _failure_ crate heavily in other parts of the project.